sys/hashes/sha3.c

```diff
@@ -0,0 +1,463 @@
+/*
+   Implementation by the Keccak, Keyak and Ketje Teams, namely, Guido Bertoni,
+   Joan Daemen, Michaël Peeters, Gilles Van Assche and Ronny Van Keer, hereby
+   denoted as "the implementer".
+
+   For more information, feedback or questions, please refer to our websites:
+   http://keccak.noekeon.org/
+   http://keyak.noekeon.org/
+   http://ketje.noekeon.org/
+
+   To the extent possible under law, the implementer has waived all copyright
+   and related or neighboring rights to the source code in this file.
+   http://creativecommons.org/publicdomain/zero/1.0/
+
+   RIOT OS adaptations (c) Mathias Tausig
+
+   This software is released under the Creative Commons CC0 1.0 license. To the extent possible under law, the implementer has waived all copyright and related or neighboring rights to the source code in this file. For more information see: http://creativecommons.org/publicdomain/zero/1.0/
```

Comment:
Some linebreaks would be nice ;-).

Reply:
Will it work with line breaks? It's just a copy/paste from the line emmanuel put into the travis config

Reply:
It shouldn't otherwise the pattern needs fixing ;-)

Reply:
Ping?

Reply:
Please address!